graphics/wxWidgets-3.2/Portfile

```diff
@@ -98,12 +99,6 @@ variant compat30 description {enable wxWidgets 3.0 compatibility (will become de
 
 # notes:
 #   --enable-unicode (already default)
-#   --enable-cxx11   (figure out how to use this switch)
-
-if {[string match *clang* ${configure.cxx}]} {
```

Comment:
@michaelld: I only noticed this part now and I would like to understand why you deleted this.

(Admittedly the port currently doesn't work on systems where clang is not default, like 10.6 and earlier, or ppc in particular, but from what I understood @kencu got it working even on 10.6, again probably with clang only anyway.)